### ISSUE TITLE
PhongDirectional (DX11.Effect) Technique pin shows Gouraud

### ISSUE THREAD

**Issue by mediadog** [mrvux/dx11-vvvv#240]
I assume this is a copy and paste error and that the techniques are actually Phong.

**Comment by mrvux** [comment]
Yeah pretty much so, not a big deal in some ways, would need some diff to fix tho.

**Comment by mrvux** [comment]
I wanted to check to difff that, but actually there's no real way to make it (vvvv diff system only allows to change pin name or  default pin value, which does not fit our case (enum)).

So I guess it will stay as is, since setting red nodes everywhere does not sound too appealing)

**Comment by joreg** [comment]
hm..but note: there will be very few red pins: only if someone used the node and actually touched the Technique pin (which should be a rare case), only then the selected enum is saved and would be red if you open it with the new pack.
